kubernetes/cray-opa/Chart.yaml

```diff
@@ -23,7 +23,7 @@
 #
 apiVersion: v2
 name: cray-opa
-version: 1.25.0
+version: 1.26.0
```

Comment:
Can you make this 1.25.2 and also push the change to main?

Reply:
Yes to both.